daemon/config_experimental.go

```diff
@@ -1,9 +0,0 @@
-// +build experimental
-
-package daemon
-
-import flag "github.com/docker/docker/pkg/mflag"
-
-func (config *Config) attachExperimentalFlags(cmd *flag.FlagSet, usageFn func(string) string) {
-	cmd.StringVar(&config.DefaultNetwork, []string{"-default-network"}, "", usageFn("Set default network"))
```

Comment:
we could. also, we can still provide user configurable default-network once we have a proper daemon level state synchronization.